pkg/api/azenvtypes.go

```diff
@@ -70,7 +70,7 @@ var (
 		TillerImageBase:                  "gcr.io/kubernetes-helm/",
 		ACIConnectorImageBase:            "microsoft/",
 		NVIDIAImageBase:                  "nvidia/",
-		AzureCNIImageBase:                "containernetworking/",
+		AzureCNIImageBase:                "mcr.microsoft.com/containernetworking/",
```

Comment:
is this available in Mooncake and other sovereign clouds?

Reply:
Not yet, it's being rolled out.

Reply:
So this PR shouldn't be merged until it's rolled out?

Reply:
I think it's OK to merge. https://github.com/Azure/aks-engine/blob/f518126807a5f05ed7633eac302fb52a496e8213/pkg/api/azenvtypes.go#L233 has the Mooncake specific URL.